www/rovercode.ini

```diff
@@ -0,0 +1,11 @@
+[uwsgi]
+module = wsgi:app
+
+master = true
+processes = 5
+
+socket = /var/www/rovercode/rovercode.sock
+chmod-socket = 777
```

Comment:
I'm not sure that it needs the execute permission and do you want it to be accessed by anyone?